From 8de2038c50527d4c3f6ea649383214b63ebc0904 Mon Sep 17 00:00:00 2001
From: Frederic Pillon <frederic.pillon@st.com>
Date: Fri, 18 Jul 2025 14:15:20 +0200
Subject: [PATCH 6/6] chore: add new field ATT bearers

Signed-off-by: Frederic Pillon <frederic.pillon@st.com>
---
 src/utility/HCISharedMemTransport.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/utility/HCISharedMemTransport.cpp b/src/utility/HCISharedMemTransport.cpp
index 1544871..6821493 100644
--- a/src/utility/HCISharedMemTransport.cpp
+++ b/src/utility/HCISharedMemTransport.cpp
@@ -665,7 +665,8 @@ int HCISharedMemTransportClass::stm32wb_start_ble(void)
     CFG_BLE_TX_PATH_COMPENS,
     CFG_BLE_RX_PATH_COMPENS,
     CFG_BLE_CORE_VERSION,
-    CFG_BLE_OPTIONS_EXT
+    CFG_BLE_OPTIONS_EXT,
+    CFG_BLE_MAX_ADD_EATT_BEARERS
   };
   /**
    * Starts the BLE Stack on CPU2
-- 
2.34.1

